.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 coverage
+.idea
```

Comment:
How can I enable this feature? I need that awesomplete ignore ñ - á - é - í - ó - ú, etc